### DISCUSSION TITLE
Load Modules

### DISCUSSION THREAD

**Discussion by ghost** [alicebob/miniredis#319] [Q&A]
Does miniredis currently support loading modules for custom commands?

**Comment by alicebob** [comment]
On Wed, Mar 29, 2023 at 01:44:35PM -0700, Justin wrote:
> Does miniredis currently support loading modules for custom commands?

Currently it doesn't. I'm not planning to add support for modules either, that
sounds too hard to get correct enough to be useful for tests (at least,
that's what I expect).
